tests/combiners_test.py

```diff
@@ -224,6 +224,12 @@ def test_mechanism(self, mechanism_type: pipeline_dp.MechanismType,
                                expected_noise_param,
                                delta=1e-3)
 
+    def test_explain_computation(self):
+        combiner = self._create_combiner(no_noise=False)
+        expected = ("Computed DP count.*\n.*Gaussian mechanism:  "
+                    "parameter=15.8.*eps=1.0  delta=1e-05.*l2_sensitivity=4.2")
+        self.assertRegex(combiner.explain_computation()(), expected)
```

Comment:
How does the double function call work exactly?

Reply:
`explain_compuation()` returns a function which returns string with explanation.